src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -599,4 +605,29 @@ private String getElementSrcWithRetryAfterWaitForPresence(By by) {
         }
     }
 
+    public void loadResultPanel(int panelId) {
+        String panel = "panelHeading-" + panelId;
+        clickAutoAjaxLoadedPanelAndWaitForExpansion(panel);
+    }
+
+    public void loadResultSectionPanel(int panelId, int sectionId) {
+        String panel = "panelHeading-section-" + panelId + "-" + sectionId;
+        clickAutoAjaxLoadedPanelAndWaitForExpansion(panel);
+    }
+
+    public void loadResultSubmitAjaxPanel(int panelId) {
+        String panel = "panelHeading-" + panelId;
+        clickSubmitAjaxLoadedPanelAndWaitForExpansion(panel);
+    }
```

Comment:
Add comments to public methods?
Do we need to use Ajax in the method name (Ajax is not a term in user's language)?